bi_view_editor/README.rst

```diff
@@ -0,0 +1,93 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+BI View Editor
+==============
+
+BI View Editor is a tool integrated in Odoo that allows users define and
+execute their own reports without the need to code.
+
+Purpose:
+
+* The BI View Editor is used to create reports not already contained in the
+  standard Odoo, combining data from existing sources.
+
+* It has been designed to be used by users with little or no knowledge of
+  the technical architecture of Odoo. Users visually link business objects
+  and select the fields to visualize.
+
+* The BI View Editor offers users different types of representations,
+  including tree, graph, pivot views.
+
+
+Usage
+=====
+
+
+To graphically design your analysis data-set:
+
+- From the Reporting menu, select "Custom BI Views"
+- Browse trough the business objects in the Query tab
+- Pick the interesting fields (Drag & Drop)
+- For each selected field, right-click on the Options column and select whether it's a row, column or measure
+- Save and click "Generate BI View"
+- Click "Open BI View" to view the result
+- If module Dashboard (board) is installed, the standard "Add to My Dashboard" functionality would be available
+- Click "Create a menu" to create a new menu item directly linked to your new BI view (this feature is available in developer mode)
```

Comment:
When the BI view is reset back to draft this menu will be removed, and you will need to re-create the menu entry.